GOVERNANCE.md

```diff
@@ -0,0 +1,81 @@
+# Abstract
+
+The purpose of this document is to formalize the governance process used by the
+tskit-dev community for core software projects, to clarify who has what responsibilities,
+and how decisions are made on changes to those responsibilities.
+
+tskit-dev is an open and inclusive community. Anyone with an interest in the
+software and science can contribute discussion, and propose changes to code, documentation
+or website content.
+
+# Scope
+
+The governance model described in this document applies to the core
+repositories of the tskit software ecosystem. These are:
+
+- [tskit - C and Python API](http://github.com/tskit-dev/tskit)
+- [msprime - coalescent simulator](http://github.com/tskit-dev/msprime)
+- [tszip - compression library for tskit](http://github.com/tskit-dev/tszip)
+- [tskit-site - website for the ecosystem](http://github.com/tskit-dev/tskit-site)
+- [kastore - key-value store](http://github.com/tskit-dev/kastore)
+- [administrative](http://github.com/tskit-dev/administrative)
+- [.github - common config](http://github.com/tskit-dev/.github)
+
```

Comment:
Great to have this laid out. Do we want explicitly say the other repos on the tskit-dev workspace (e.g. tscompress, tsinfer) may become the responsibility of other maintainers if need be, or is that too prescriptive?

Reply:
I thought about this, but decided, as they are out of scope for this document, that it shouldn't say anything about them.